recipes/mlio/conda_build_config.yaml

```diff
@@ -0,0 +1,10 @@
+# The default deployment target used by conda-forge is 10.9 which is not
+# compatible with pybind11 due to lack of sized C++ deallocation.
```

Comment:
pybind11 works fine with 10.9

Reply:
With C++17 pybind11 uses a C++17 specific [deallocation function](https://github.com/pybind/pybind11/blob/master/include/pybind11/pybind11.h#L1008). Clang enables these "sized deallocation" functions only when the `-fsized-deallocation` compiler option is specified. However enabling this option causes the regular `operator delete()` to fail with the SDK v10.9.